src/components/ErrorText/ErrorText.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+
+import { ErrorText } from './ErrorText';
+
+describe('ErrorText', () => {
+  it('renders with default ErrorText', () => {
+    render(<ErrorText>Error message</ErrorText>);
+    const error = screen.getByRole('paragraph');
+    expect(error).toBeInTheDocument();
+    expect(error).toHaveTextContent(/Error message/i);
+  });
+  it('renders with custom styles', () => {
```

Comment:
Again no whitespace separating tests, every test should be separated out, it just makes code folding possible, as it is right now, I cannot fold the code to focus on a single test.

We should look into automating this, so the developer cannot forget, which is easily done. Here is the list of vitest rules https://github.com/vitest-dev/eslint-plugin-vitest